### PR TITLE
more code and documentation for group characters

### PR DESCRIPTION
**Pull request by ThomasBreuer**

### CODE REVIEW

File: src/Groups/GAPGroups.jl [hunk 1]
```diff
@@ -889,29 +891,95 @@ end
...
+julia> issimple(symmetric_group(5))
+false
+
+```
 """
 @gapattribute issimple(G::GAPGroup) = GAP.Globals.IsSimpleGroup(G.X)::Bool
 
 @doc Markdown.doc"""
-    isalmostsimple(G)
+    isalmostsimple(G::GAPGroup)
```
Comment (fingolfin) [inline src/Groups/GAPGroups.jl:929]: We could rename it to `is_almostsimple` -- or perhaps even `is_almost_simple` (?) now (as it is "almost simple group", unlike "quasisimple group"...)

Comment (fingolfin) [reply src/Groups/GAPGroups.jl:929]: But of course it's fair to leave this up to a "big rename" which we could do next week

File: src/Groups/group_characters.jl [hunk 2]
```diff
@@ -275,8 +286,49 @@ end
...
+        push!(L1, GAP.julia_to_gap(L[i]))
+        hadfunc = false
+      end
+    end
+
+    if ordered_by isa Function
+      K = GAP.call_gap_func(GAP.Globals.AllCharacterTableNames, L1...;
+            OrderedBy = find_index_function(ordered_by, false)[2])
+    else
+      K = GAP.call_gap_func(GAP.Globals.AllCharacterTableNames, L1...)
```
Comment (fingolfin) [inline src/Groups/group_characters.jl:330]: Doesn't this work, too?
```suggestion
      K = GAP.Globals.AllCharacterTableNames(L1...)
```

File: src/Groups/group_characters.jl [hunk 3]
```diff
@@ -651,6 +814,75 @@ function decomposition_matrix(modtbl::GAPGroupCharacterTable)
...
+ [1, 2, 3, 6, 7]
+ [1, 2, 3, 7, 6]
+ [1, 2, 4, 6, 7]
+ [1, 2, 4, 7, 6]
+
+```
+"""
+function possible_class_fusions(subtbl::GAPGroupCharacterTable, tbl::GAPGroupCharacterTable)
+  fus = GAP.Globals.PossibleClassFusions(subtbl.GAPTable, tbl.GAPTable)::GAP.GapObj
+  return [Vector{Int}(x) for x in fus]
```
Comment (fingolfin) [inline src/Groups/group_characters.jl:884]: This is a minor detail, but Julia won't know the type of `x`; it seems to me it is always a GAP list here. If so, a type assertion will allow Julia to deduce which `Vector{Int}` constructor is invoked during precompile time, and hence also deduce the return type of this function (i.e. make it type stable)
```suggestion
  return [Vector{Int}(x::GapObj) for x in fus]
```

**Review by fingolfin — APPROVED** [review]

File: src/Groups/group_characters.jl (hunk 3 above)
Comment (ThomasBreuer) [reply src/Groups/group_characters.jl:884]: Yes.

File: src/Groups/group_characters.jl (hunk 2 above)
Comment (ThomasBreuer) [reply src/Groups/group_characters.jl:330]: Yes.

File: src/Groups/GAPGroups.jl (hunk 1 above)
Comment (ThomasBreuer) [reply src/Groups/GAPGroups.jl:929]: As a rule of thumb, I have used underscores when introducing new function names, but I have not renamed existing functions.

**Pull request merged**